src/Uno.UI/UI/Xaml/UIElement.cs

```diff
@@ -829,7 +829,16 @@ private static void InnerUpdateLayout(UIElement root)
 				else if (root.IsArrangeDirtyOrArrangeDirtyPath)
 				{
 					root.Arrange(bounds);
+#if !IS_UNIT_TESTS
+					root.XamlRoot.RaiseInvalidateRender();
```

Comment:
Weird to see it here, if it's a wrokaround please add a comment :)

Reply:
Yes it's a workaround for a TextBox issue (not sure if it's a bug on TextBox side). I added a comment.